server/models/idea.js

```diff
@@ -0,0 +1,45 @@
+const MongoClient = require('mongodb').MongoClient;
+const url = 'mongodb://localhost:27017/rankerdb';
+
+exports.all = function(callback) {
+	MongoClient.connect(url, function(err, db) {
```

Comment:
We probably don't want to be establishing a new connection to MongoDB on every single request. Maybe before the server starts listening, create a connection and reuse that connection for each of these routes.